nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -12,6 +12,40 @@ These release notes track the development of the {secondary-scheduler-operator-f
 
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.1.0"]
+== Release notes for {secondary-scheduler-operator-full} 1.1.0
+
+// TODO: Update date if necessary
+Issued: 2022-8-15
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.1.0:
+
+* link:https://access.redhat.com/errata/RHSA-2022:TODO[RHSA-2022:TODO]
+
+// TODO: Update advisory link
+// https://errata.devel.redhat.com/advisory/97046
+
+[id="secondary-scheduler-operator-1.1.0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* TODO
+// Need anything related to this?
+// * The maximum {product-title} version for {secondary-scheduler-operator-full} 1.1.0 is 4.11.
```

Comment:
@ingvagabund We pointed out the max version for v1.0.1. Is there anything new/different that we need to point out related to this for v1.1?